tests/util/keyring.py

```diff
@@ -165,7 +165,7 @@ def _patch_and_create_keychain(
             add_dummy_key_to_cryptfilekeyring(crypt_file_keyring)
 
         keychain = Keychain(user=user, service=service)
-        keychain.keyring_wrapper = KeyringWrapper(keys_root_path=Path(temp_dir))
+        keychain.keyring_wrapper = KeyringWrapper.create(keys_root_path=Path(temp_dir))
```

Comment:
```suggestion
        keychain.keyring_wrapper = KeyringWrapper.create(keys_root_path=Path(temp_dir))
        assert keyring_wrapper is not None
        keychain.keyring_wrapper = keyring_wrapper
```

Reply:
this is to fix the mypy error (untested):
```
tests/util/keyring.py:168: error: Incompatible types in assignment (expression has type "Optional[KeyringWrapper]", variable has type "KeyringWrapper")  [assignment]
```

My understanding is that this is how you unwrap optionals in python

Reply:
`.create()` is only optional by mistake I think.  https://github.com/Chia-Network/chia-blockchain/pull/10358#discussion_r820005241  If you are sprinkling `assert`s around for `None`s, maybe you didn't want the thing to be optional.  Also see the keybase discussion about `assert`s.  `:]`

Reply:
I agree that `create()` shouldn't return an `Optional`, but assuming the existing code in this patch is correct, this would be the correct use of assert. The existing code clearly expects it to never return `None` in this case. If it did, it would be a programmer mistake, not an expected/recoverable failure.

Reply:
Thanks guys.